pkg/config/config_template.yaml

```diff
@@ -127,6 +127,18 @@ api_key:
 # number of check runners instances concurrently. Setting the value to 1
 # would result in checks running sequentially.
 #
+# NOTE: due to the nature of the python interpreter and it's global interpreter
+# lock (GIL) only one native thread may be running on the python interpreter at
+# a time. The introduction of concurrent check runners implies concurrently
```

Comment:
Let's remove `introduction of` here (only makes sense in the context of an upgrade to 6.5)